src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -281,6 +281,7 @@ void InitIntConvCoeff(EnergyPlusData &state,
         for (int spaceNum : zone.spaceIndexes) {
             auto const &thisSpace = state.dataHeatBal->space(spaceNum);
             for (int SurfNum = thisSpace.HTSurfaceFirst; SurfNum <= thisSpace.HTSurfaceLast; ++SurfNum) {
+                Real64 spaceMAT = state.dataZoneTempPredictorCorrector->spaceHeatBalance(spaceNum).MAT;
```

Comment:
Shouldn't you move this up above the for loop? This is reading the same space MAT for each surface.